### PR TITLE
docs: fix doc-bot style during load

### PR DESCRIPTION
**Pull request by nomagick**
Fix doc-bot style broken before load.
Fix potential issue that may cause doc-bot fail to load.

### CODE REVIEW

File: docs/_static/docbot.js [hunk 1]
```diff
@@ -1,100 +1,104 @@
...
-        general_config: {
-            server_address: server_address || 'https://docsbot.jina.ai',
-            search_endpoint: '/search',
-            slack_endpoint: '/slack'
+$(document).ready(function () {
+    const app = new Vue({
+        el: '#jina-docbot',
+        delimiters: ['${', '}'],
+        data: {
+            ready: true,
```
Comment (nomagick) [inline docs/_static/docbot.js:6]: Added ready state, which is always `true` from the Vue point of view.

File: docs/_static/docbot.js [hunk 2]
```diff
@@ -1,100 +1,104 @@
...
-    el: '#jina-docbot',
-    delimiters: ['${', '}'],
-    data: {
-        is_busy: false,
-        is_conn_broken: false,
-        general_config: {
-            server_address: server_address || 'https://docsbot.jina.ai',
-            search_endpoint: '/search',
-            slack_endpoint: '/slack'
+$(document).ready(function () {
```
Comment (nomagick) [inline docs/_static/docbot.js:1]: Wrapped all inside `document.ready`.

Not many changes in this file

**Comment by JoanFM** [comment]
Hey @nomagick can u add some screenshots of the previous state and new state?

**Comment by nomagick** [comment]
> Hey @nomagick can u add some screenshots of the previous state and new state?
Sure @JoanFM 

Before:
截屏2021-11-19 17 14 54

After:
截屏2021-11-19 16 47 08

**Pull request merged**